test/Conversion/TorchToMhlo/basic.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: torch-mlir-opt <%s -convert-torch-to-mhlo -split-input-file -verify-diagnostics | FileCheck %s
+// RUN: torch-mlir-opt%exeext <%s -convert-torch-to-mhlo -split-input-file -verify-diagnostics | FileCheck %s
```

Comment:
This should not be necessary, lit supports having this "tool" selected correctly on all platforms. Have a look at how `mlir-opt` is handled, for example.